model/Image.php

```diff
@@ -711,7 +711,7 @@ public function cacheFilename($format) {
 		array_shift($args);
 		$folder = $this->ParentID ? $this->Parent()->Filename : ASSETS_DIR . "/";
 
-		$format = $format . base64_encode(json_encode($args, JSON_NUMERIC_CHECK));
+		$format = $format . base64_encode(json_encode(array_map('strval', $args)));
```

Comment:
http://php.net/manual/en/json.constants.php says: "Encodes numeric strings as numbers.". So I guess this should be `intval` rather than `strval`?